Doc/library/importlib.rst

```diff
@@ -420,30 +420,53 @@ ABC hierarchy::
         reloaded):
 
         - :attr:`__name__`
-            The name of the module.
+            The module's fully-qualified name.
+            It is ``'__main__'`` for non-package modules run from the file
```

Comment:
This part about `__main__` certainly adds value, but should be in [the `__name__` subsection of the language reference](https://docs.python.org/3/reference/import.html#__name__).  The line above, `The module's fully-qualified name.`, is a nice clarification.

FWIW, the text is a bit unclear and seems to say something that isn't correct.

Reply:
Alright.